stories/Header/CompleteHeader/basic.tsx

```diff
@@ -135,14 +135,17 @@ export class SlimHeader extends Component<
 }
 
 export class CenterHeader extends Component<
-  { theme?: 'light' | 'dark' | ''; iconName: string },
+  { theme?: 'light' | 'dark' | ''; iconName: string; iconAlt?: string },
```

Comment:
Can you pass an alt string also here to use this prop? https://github.com/italia/design-react-kit/pull/864/files#diff-5cd1ad6642e2f768061589327cb5b6e68233f59a39f80814afbc11b816a6c22aL250